eng/install-native-dependencies.sh

```diff
@@ -13,37 +15,35 @@
 #
 # ./install-native-dependencies.sh <OS>
 
-if [ "$1" = "Linux" ]; then
-    sudo apt update
-    if [ "$?" != "0" ]; then
-       exit 1;
-    fi
-    sudo apt install cmake llvm-3.9 clang-3.9 lldb-3.9 liblldb-3.9-dev libunwind8 libunwind8-dev gettext libicu-dev liblttng-ust-dev libcurl4-openssl-dev libssl-dev libkrb5-dev libnuma-dev build-essential
-    if [ "$?" != "0" ]; then
-        exit 1;
-    fi
-elif [[ "$1" == "MacCatalyst" || "$1" == "OSX" || "$1" == "tvOS" || "$1" == "iOS" ]]; then
-    engdir=$(dirname "${BASH_SOURCE[0]}")
+os="$(echo "$1" | tr "[:upper:]" "[:lower:]")"
+
+if [ -e /etc/os-release ]; then
+    . /etc/os-release
+fi
 
-    echo "Installed xcode version: `xcode-select -p`"
+if [ "$os" = "linux" ] && { [ "$ID" = "debian" ] || [ "$ID_LIKE" = "debian" ]; }; then
+    apt update
+
+    apt install -y build-essential gettext locales cmake llvm clang lldb liblldb-dev libunwind8-dev libicu-dev liblttng-ust-dev \
+        libssl-dev libkrb5-dev libnuma-dev libz-dev
```

Comment:
the [docs](https://github.com/dotnet/runtime/blob/main/docs/workflow/requirements/linux-requirements.md#toolchain-setup) suggest zlib1g-dev instead:
 
```suggestion
        libssl-dev libkrb5-dev libnuma-dev zlib1g-dev
```

Reply:
[libz-dev](https://packages.ubuntu.com/jammy/libz-dev) installs zlib1g-dev. i like libz-dev because it shares naming convention with every other debian dev packages.